src/api/stats/fantom/getHectorApy.ts

```diff
@@ -0,0 +1,52 @@
+const { fantomWeb3: web3 } = require('../../../utils/web3');
+const { FANTOM_CHAIN_ID: chainId } = require('../../../constants');
+import { getEDecimals } from '../../../utils/getEDecimals';
+import { getRewardPoolApys } from '../common/getRewardPoolApys';
+
+import { addressBook } from '../../../../packages/address-book/address-book';
+const {
+  fantom: {
+    tokens: { WFTM },
+  },
+} = addressBook;
+
+const singlePool = [
+  {
+    name: 'curve-ftm-tor',
```

Comment:
please rename to hector-something as its not curve farm

Reply:
Does this apply to the entry in `src/data/fantom/curvePools.json` as well? I tried removing it, but that seems to break things. 

Also, for instance, `curve-ftm-mim` is named this way too even though it is staked on abracadabra.money, not Curve.

Reply:
> Also, for instance, `curve-ftm-mim` is named this way too even though it is staked on abracadabra.money, not Curve.

no it's not